app/controllers/users_controller.rb

```diff
@@ -4,34 +4,36 @@ class UsersController < ApplicationController
 
   def index
     authorize User
+    skip_policy_scope
 
     @pending = policy_scope(User).pending.recent_first.page(params[:pending_page]).per(3)
 
-    @status = if params[:status].present?
-                if params[:status] == 'All'
-                  nil
-                else
-                  UserStatus.find(params[:status])
-                end
-              else
-                UserStatus.default_active_status
-              end
+    filters = [{ input_label: 'Status',
+                 column: :user_status_id,
+                 options: UserStatus.not_pending.by_id,
+                 default_value: UserStatus.default_active_status.id }]
 
-    @users = policy_scope(User).recent_first
-    @users = @users.where(user_status: @status) if @status
-    @users = @users.where(external: params[:external]) if params[:external].present?
+    if RadCommon.external_users
+      filters.push({ input_label: 'Type', column: :external, scope_values: %i[internal external] })
```

Comment:
@elbzero - why is `column` required for this type of search filter?

Reply:
@elbzero ^

Reply:
@garyfoster I believe its because we had a situation where we wanted to mix scope values with regular values for example: Users Unassigned, John, Joe, so we need to know what column to filter when its not a 'scope value' we can relax that constraint if needed.